indico/MaKaC/schedule.py

```diff
@@ -35,6 +35,7 @@
                                     IAttachmentFossil, IFolderFossil)
 from MaKaC.common.cache import GenericCache
 from indico.util.decorators import classproperty
+from flask import g
```

Comment:
The imports are already well randomized and the content of this file is going away soon. (See comment below.)